app/src/main/java/com/example/alexandraboukhvalova/a436msclinicalmonitoring/instruction.java

```diff
@@ -0,0 +1,29 @@
+package com.example.alexandraboukhvalova.a436msclinicalmonitoring;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+
+public class instruction extends Activity {
```

Comment:
Change the classname to something descriptive, preferably "TappingInstructionsActivity"